salt/_pillar/metalk8s.py

```diff
@@ -37,9 +39,30 @@ def _load_networks(config_data):
     }
 
 
+def _get_labels(config="/etc/kubernetes/admin.conf", context="kubernetes-admin@kubernetes"):
+    # TODO: Use the "bootstrap_config" instead of /etc/kubernetes/admin.conf
+    labels = {}
+
+    hostname = __grains__.get('localhost')
+    if not hostname or not os.path.isfile(config):
+        return labels
+
+    try:
+        labels = __salt__['kubernetes.node_labels'](
+            name=hostname,
+            kubeconfig=config,
+            context=context
+        )
+    except Exception as exc:  # pylint: disable=broad-except
```

Comment:
Why `Exception`, seems like the code only raises `CommandExecutionError`, no?

Reply:
Don't want to break the whole external pillar if we got a ValueError, IndexError, NoneType or what ever

Reply:
Fair enough.